 crazys-wm-mod/cDungeon.cpp

```diff
@@ -408,11 +408,21 @@ void sDungeonGirl::OutputGirlDetailString(string& Data, const string& detailName
 	else if (detailName == "Duration")			{ ss << m_Weeks; }
 	else if (detailName == "Feeding")			{ ss << ((m_Feeding) ? gettext("Yes") : gettext("No")); }
 	else if (detailName == "Tortured")			{ ss << ((m_Girl->m_Tort) ? gettext("Yes") : gettext("No")); }
+	else if (detailName == "Kidnapped")			
+	{
+		int duration = cGirls::HasTempTrait(m_Girl, "Kidnapped");
+

```

Comment:
Although I fail to see what kind of trouble the STATIC could cause there I agree that yours is a better solution.